src/components/Loader/loader.css

```diff
@@ -0,0 +1,33 @@
+.wrapper {
```

Comment:
Please, use material styling and rename file to `loader.styles.js` as much as possible and avoid using the *.css` files

Reply:
used